### DISCUSSION TITLE
Controlling 48 servos

### DISCUSSION THREAD

**Discussion by TomWaterz** [ArminJo/ServoEasing#80] [Q&A] [answered]
In the documentation it says "ESP32 the I2C library is only capable to run at 100 kHz" and not to use more than 2 expander boards. What microcontroller would you recommend to move 48 servos all at the same time? and is it possible with this library?

**Comment by ArminJo** [comment, marked as answer]
Teensy or BluePill or one of the fast (>16 MHz) Arduino boards, but dependent on the workload, even a Uno may be sufficient.

**Comment by TomWaterz** [comment]
gotcha. ill grab a teensy 4.0 and try it out! thanks for the library. appreciate all the work.
